sympy/utilities/lambdify.py

```diff
@@ -1059,8 +1192,10 @@ def __init__(self, printer=None, dummify=False):
         # Used to print the generated function arguments in a standard way
         self._argrepr = LambdaPrinter().doprint
 
-    def doprint(self, funcname, args, expr):
-        """Returns the function definition code as a string."""
+    def doprint(self, funcname, args, expr, *, useless_symbols=[], replacements=[]):
```

Comment:
Use `()` instead of `[]` for default arguments. 

```suggestion
    def doprint(self, funcname, args, expr, *, useless_symbols=(), replacements=()):
```

Reply:
Let's also make these start with an underscore. This API is fine for now, but eventually we should replace it with codegen AST as I described in my other comment.